src/recipe/recipe.controller.ts

```diff
@@ -116,5 +132,6 @@ export class RecipeController {
   @Delete(':id')
   async deleteRecipe(@Param('id', ParseIntPipe) id: number): Promise<void> {
     await this.recipeService.deleteRecipe(id);
+    await this.redisCacheService.del(`cache:recipe:${id}`);
```

Comment:
This is the 4th time we are repeating the same string (cache key) - that's why I recommend moving it to a dedicated cache service and perhaps creating a private method that will return it